homeassistant/components/wallbox/const.py

```diff
@@ -50,49 +54,57 @@ class WallboxSensorEntityDescription(SensorEntityDescription):
         precision=2,
         native_unit_of_measurement=POWER_KILO_WATT,
         device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     CONF_MAX_AVAILABLE_POWER_KEY: WallboxSensorEntityDescription(
         key=CONF_MAX_AVAILABLE_POWER_KEY,
         name="Max Available Power",
         precision=0,
         native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
         device_class=DEVICE_CLASS_CURRENT,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     CONF_CHARGING_SPEED_KEY: WallboxSensorEntityDescription(
         key=CONF_CHARGING_SPEED_KEY,
         icon="mdi:speedometer",
         name="Charging Speed",
         precision=0,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     CONF_ADDED_RANGE_KEY: WallboxSensorEntityDescription(
         key=CONF_ADDED_RANGE_KEY,
         icon="mdi:map-marker-distance",
         name="Added Range",
         precision=0,
         native_unit_of_measurement=LENGTH_KILOMETERS,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
     ),
     CONF_ADDED_ENERGY_KEY: WallboxSensorEntityDescription(
         key=CONF_ADDED_ENERGY_KEY,
         name="Added Energy",
         precision=2,
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
         device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
     ),
     CONF_CHARGING_TIME_KEY: WallboxSensorEntityDescription(
         key=CONF_CHARGING_TIME_KEY,
         icon="mdi:timer",
         name="Charging Time",
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     CONF_COST_KEY: WallboxSensorEntityDescription(
         key=CONF_COST_KEY,
         icon="mdi:ev-station",
         name="Cost",
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
costs seem an aggregation, thus not a measurement?

Reply:
true, that would also make it total_increasing (check?) it increases during a session and then resets to zero

Reply:
and also done